### PR TITLE
feat(app): add module calibration section to robotsettings

### PR DESCRIPTION
**Pull request by koji**
# Overview
Add Module calibration section to RobotSettings Calibration tab
close RAUT-556

OverflowMenu doesn't do anything since Wizard isn't implemented and no clear data function in BE.
These will be implemented in following PRs.
This PR includes clear data confirmation modal but as I said there is no clear data function so I added a Storybook to check the UI. 

Screenshot 2023-08-23 at 2 04 50 PM
Screenshot 2023-08-23 at 2 05 17 PM
Screenshot 2023-08-23 at 2 05 28 PM
Screenshot 2023-08-23 at 2 05 41 PM

Screenshot 2023-08-24 at 10 57 18 AM

# Test Plan
Attached modules and open Desktop app and go to RobotSettings

# Changelog
- add components and tests for ModuleCalibration 
- add confirmation modal

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] call wizard

File: app/src/assets/localization/en/device_settings.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
   "calibrate_deck_to_dots": "Calibrate deck to dots",
   "calibrate_deck": "Calibrate deck",
   "calibrate_gripper": "Calibrate gripper",
+  "calibrate_module": "Calibrate module ",
```
Comment (b-cooper) [inline app/src/assets/localization/en/device_settings.json]: is this trailing space intentional?

File: app/src/assets/localization/en/device_settings.json [hunk 2]
```diff
@@ -195,6 +203,7 @@
   "protocol_run_history": "Protocol run History",
   "recalibrate_deck": "Recalibrate deck",
   "recalibrate_gripper": "Recalibrate gripper",
+  "recalibrate_module": "Recalibrate module ",
```
Comment (b-cooper) [inline app/src/assets/localization/en/device_settings.json]: another suspicious trailing space here

File: app/src/organisms/RobotSettingsCalibration/CalibrationDetails/ModuleCalibrationOverflowMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          closeFlow={() => {
+            setShowModuleWizard(false)
+          }}
+        />
+      ) : null}
+      {showOverflowMenu ? (
+        <Flex
+          ref={OverflowMenuRef}
+          whiteSpace="nowrap"
+          zIndex={10}
```
Comment (b-cooper) [inline app/src/organisms/RobotSettingsCalibration/CalibrationDetails/ModuleCalibrationOverflowMenu.tsx]: it looks like this `zIndex` might be higher than the wizard's overlay.  The screenshot you attached shows the overflow menu higher in the z than the overlay.  It should be lower

File: app/src/assets/localization/en/device_settings.json (hunk 1 above)
Comment (koji) [reply app/src/assets/localization/en/device_settings.json]: nope, I'll remove that.

File: app/src/organisms/RobotSettingsCalibration/CalibrationDetails/ModuleCalibrationOverflowMenu.tsx (hunk 3 above)
Comment (koji) [reply app/src/organisms/RobotSettingsCalibration/CalibrationDetails/ModuleCalibrationOverflowMenu.tsx]: reduce zIndex to 5 and added  `setShowOverflowMenu(false)` to dismiss the overflowMenu when tapping calibrate button.

https://github.com/Opentrons/opentrons/assets/474225/8cb50e55-4f36-4b89-a936-d4c9ead8624a

**Review by b-cooper — APPROVED** [review]
👾

**Pull request merged**